api/tests/Feature/PoolTest.php

```diff
@@ -204,6 +241,8 @@ public function testPoolQueryReturnsOnlyPublished(): void
         $draftPool = Pool::factory()->create([
             'published_at' => null,
         ]);
+        // this pool is archived so it should not be returned in the publishedPool query
```

Comment:
You can leave this here, but I'd recommend making a separate test to check that archived pools aren't returned in the main query. Having an atomic test like that would make it more obvious what's happening if it failed.

Reply:
Yeah, smaller units tests are better.  I've just split the test in half.
[split test into two](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7399/commits/9959bc4c71920a2a339dd4845054caabd381b43d)

Reply:
And [split anon test](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7399/commits/a62be9da70456ff762b6d6f670102a59f45a5743)